typescript/src/app/newcomp/newcomp.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { NewcompComponent } from './newcomp.component';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/newsbridge/sandbox/pullRequest?prid=7539937)
```suggestion
import { NewcompComponent } from "./newcomp.component";
```